docs/process-promise.md

```diff
@@ -14,6 +14,23 @@ const p = $({halt: true})`command`
 const o = await p.run()
 ```
 
+## `stage`
+
+Returns current internal state machine stage
```

Comment:
The shorter the better.

Shows the current process stage: 'initial' | 'halted' | 'running' | 'fulfilled' | 'rejected'

```ts
const p = $`echo foo`
p.stage // 'running'
await p
p.stage // 'fulfilled'
```